libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -49,10 +49,37 @@ do {                                            \
                        OUTOFGEOFENCE)
 
 
+static uint8_t counter = 0;     // For Debuging
+
+
 AP_GPS_SBF::AP_GPS_SBF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
-                       AP_HAL::UARTDriver *_port) :
+                       AP_HAL::UARTDriver *_port, uint8_t _asterx_setup) :
     AP_GPS_Backend(_gps, _state, _port)
 {
+    switch(_asterx_setup) {
+        case 0:
+            _asterx_type_is_singleantenna = true;
+            _asterx_type_is_i = false;
+            _asterx_type_is_dualantenna = false;
+            gcs().send_text(MAV_SEVERITY_INFO, "SBF Single Antenna\n");
+            break;
+        case 1:
+            _asterx_type_is_singleantenna = false;
+            _asterx_type_is_i = true;
+            _asterx_type_is_dualantenna = false;
+            gcs().send_text(MAV_SEVERITY_INFO, "SBF INS\n");
+            break;
+        case 2:
+            _asterx_type_is_singleantenna = false;
+            _asterx_type_is_i = false;
+            _asterx_type_is_dualantenna = true;
+            gcs().send_text(MAV_SEVERITY_INFO, "SBF Dual Antenna\n");
+            break;
+        default:
```

Comment:
Force it into single antenna mode in this case.

Reply:
Ok